spec/expression.dd

```diff
@@ -1671,19 +1672,21 @@ $(GNAME AssertExpression):
 
     $(P When compiled and run, it will produce the message:)
 
-$(CONSOLE
-core.exception.AssertError@test.d(3) an error message
-)
+    $(CONSOLE core.exception.AssertError@test.d(3) an error message)
+
+    The implementation may handle $(D assert(0)) differently in that in release mode
+    it may simply generate a $(D HLT) instruction or equivalent, even
+    in release mode.
```

Comment:
Repetition of "in release mode".

Reply:
@aG0aep6G you may want to make obvious/noncontroversial fixes like this by editing the PR and committing to it directly. It works real nice... props for the github team.

Reply:
> @aG0aep6G you may want to make obvious/noncontroversial fixes like this by editing the PR and committing to it directly. It works real nice... props for the github team.

I don't think I have the powers to do that. I'm not a member of Team D or anything.

Reply:
@aG0aep6G if you go to "files", do you see something like a crayon in the decoration at the top of each file changed?

Reply:
> @aG0aep6G if you go to "files", do you see something like a crayon in the decoration at the top of each file changed?

Yeah. It's not clickable and the tooltip says: "You must be signed in and have push access to the AssertExp branch to make changes." I don't have that kind of access to the D repositories.